src/patterns/question-pages/index.md.njk

```diff
@@ -83,6 +83,21 @@ For example, ‘About you’
 
 You can also learn more about how starting with [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page) helps users in the GOV.UK Service Manual.
 
+#### Asking complex questions without using hint text
+
+Do not use hint text if you need to give a lengthy explanation with lists and paragraphs. Screen readers read out the entire text when users interact with the form element. This could frustrate users if the text is long.
+
+You should also avoid using links in hint text, as screen readers might not read out them out.
```

Comment:
Should this be more strongly-worded, to match the existing guidance on the Text input page?

> Avoid links
> Do not include links within hint text. While screen readers will read out the link text when describing the field, they will not tell users that the text is a link.

```suggestion
Do not use links in hint text, as while screen readers will read out the link text when describing the field, they will not tell users that the text is a link.
```

Reply:
That's mega, thanks @frankieroberto! Just giving it a tweak:

```suggestion
Do not use links in hint text. While screen readers will read out the link text when describing the field, they will not tell users the text is a link.
```